patterns/byod.md

```diff
@@ -0,0 +1,579 @@
+
+### Mobile device access (BYOD Option 1)
+
+The Mobile device access option will leverage:
+
+* Azure Active Directory Multi-Factor Authentication (MFA) and Conditional Access policies
+
+* Defender for Cloud Apps to prevent data leakage and spills
+
+* Intune - Application Protection Policies.
+
+
+The outcome will allow the end user to: 
+* Access email using Outlook on their personal Mobile Device. 
+* Access Teams on their personal Mobile Device 
+* Access SharePoint Online on their personal Mobile Device 
+* Use of Word, Excel, PowerPoint approved mobile apps.
+
+**Mobile devices** in the context of this BYOD option are smart phones or tablets running Apple iOS or Android.
+
+#### Mobile Application Management
+
+Access via Mobile Application Management (MAM)) is provided through approved applications published by Microsoft to the respective app store (Apple AppStore or Google Play Store). 
+
+* Intune app protection policies provide a container policy to the applications to secure corporate data, and prevent download upload, copy and paste actions, and enforce a PIN/appropriate passcode. 
+* A MAM model does not require the device to be managed (MDM) or enrolled so will carry additional risk. Intune app protection MAM policies can be applied without enrolment requirement, this is referred to as MAM without enrolment (MAM-WE). Approved applications can be defined through the Conditional Access policies.  Conditional Access policies can be applied based on various attributes such as device state, user group or location.
+* With MAM, iOS devices do not require Apple Business Manager or Apple Push Notification Service certificate configuration (the configuration in the Blueprint for MDM can be ignored if this pattern is the only method for Mobile Devices for the Agency). Without MDM policies and enrolment, various functions cannot be prevented such as the ability to restrict screenshot for example.
+* To protect the data in the event that a device is lost or stolen or if the employee leaves the Agency, Selective Wipe capability can be used to wipe the corporate data in managed apps from the device. Intune app protection policies can help prevent data contamination between non managed apps on a client device.
+* Agencies should provide some guidance or user guides to their users on how to install the Office applications on their devices and authenticate to avoid confusion. Note, these actions and out-of-the-box setup are generally automated within a supervised, managed deployment. 
+
+#### Azure AD Identity Protection
+
+Azure AD Identity Protection is an Azure AD feature that protects the identity through:
+
+* automation of detection and remediation of identity-based risks,
+* includes additional challenge to user based on risk detection in real time, and
+* Provides a portal for identify based investigation.
```

Comment:
`identify` is incorrect, is it meant to be `identity`? Also lower case `provides`